plugins/node/opentelemetry-instrumentation-winston/src/instrumentation.ts

```diff
@@ -138,22 +159,24 @@ export class WinstonInstrumentation extends InstrumentationBase {
         this: never,
         ...args: Parameters<typeof original>
       ) {
-        const span = trace.getSpan(context.active());
+        const config = instrumentation.getConfig();
+        const record = args[0];
 
-        if (!span) {
-          return original.apply(this, args);
+        if (!config.disableLogCorrelation) {
+          const span = trace.getSpan(context.active());
```

Comment:
nit: you can use a shortcut for that in the API:
```suggestion
          const span = trace.getActiveSpan();
```